lib/Driver/ToolChain.cpp

```diff
@@ -149,3 +149,11 @@ ToolChain::findProgramRelativeToSwiftImpl(StringRef executableName) const {
 types::ID ToolChain::lookupTypeForExtension(StringRef Ext) const {
   return types::lookupTypeForExtension(Ext);
 }
+
+bool
+ToolChain::sanitizerRuntimeLibExists(const ArgList &args,
+                                     StringRef sanitizerName) const {
+  // Lookup is currently only supported for Darwin,
+  // this method should be overriden by a subclass.
```

Comment:
Please drop this comment. You can say something like "Assume no sanitizers are supported on a particular platform by default" if you want.